hyperspyui/plugins/fft.py

```diff
@@ -57,287 +54,194 @@ def create_actions(self):
                         "navigator",
                         selection_callback=self.ui.select_signal)
 
-        self.add_action('nfft', "Signal FFT", self.nfft,
-                        icon='nfft.svg',
-                        tip="Perform a fast fourier transform on the " +
-                            "entire signal",
-                        selection_callback=self.ui.select_signal)
```

Comment:
Not useful to have the full ND FFT after all? :)

Reply:
This is what the FFT in hyperspy does.

Reply:
Ah, neat.